bundler/spec/commands/lock_spec.rb

```diff
@@ -98,6 +118,9 @@
       DEPENDENCIES
         foo
 
+      CHECKSUMS
+        #{checksum_for_repo_gem repo, "foo", "1.0", :empty => true}
```

Comment:
Is this `empty` parameter needed? Will that just lock `foo (1.0)`?

Reply:
yeah, it omits the actual checksum

Reply:
I'd remove this parameter then and check the real content for consistency, because I've seen both styles used around.

Reply:
I replaced all these with no_checksum or something similar so they are (mostly) uniform.